mllib/src/main/scala/org/apache/spark/mllib/clustering/PowerIterationClustering.scala

```diff
@@ -412,11 +434,21 @@ object PowerIterationClustering extends Logging {
    */
   private[clustering]
   def kMeans(v: VertexRDD[Double], k: Int): VertexRDD[Int] = {
-    val points = v.mapValues(x => Vectors.dense(x)).cache()
+    val points = v.mapValues(Vectors.dense(_)).cache()
```

Comment:
Not a big deal but you can remove `(_)` here and below

Reply:
I think we can't remove `(_)`, because there are multiple implementations for the methods.
https://github.com/apache/spark/blob/2f558094257c38d26650049f2ac93be6d65d6d85/mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala#L75-L94